{
    "affected": [
        {
            "package": {
                "ecosystem": "photon:2.0",
                "name": "python2",
                "purl": "pkg:rpm/vmware/python2?distro=photon-2"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "2.7.15-14.ph2"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        },
        {
            "package": {
                "ecosystem": "Photon OS:2.0",
                "name": "python2",
                "purl": "pkg:rpm/vmware/python2?distro=photon-2"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "2.7.15-14.ph2"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        }
    ],
    "id": "PHSA-2020-0211",
    "modified": "2023-05-07T05:25:00Z",
    "published": "2020-02-24T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-2.0-211"
        }
    ],
    "related": [
        "CVE-2019-9674"
    ]
}